alfasim_sdk/cli.py

```diff
@@ -89,6 +115,12 @@ def template(dst, caption, plugin_id, author_name, author_email):
 @main.command(name="compile")
 @plugin_dir_option
 def _compile(plugin_dir):
+    """
+    Compile the plugin from the given plugin-dir option, when not provided, plugin-dir will be the current folder location.
```

Comment:
Punctuation: `Compile the plugin from the given plugin-dir option. When not provided, plugin-dir`